students/sangwoong/users/views.py

```diff
@@ -56,14 +58,20 @@ def post(self, request):
             data          = json.loads(request.body)
             user_account  = data["account"]
             user_password = data["password"]
+            user_id       = User.objects.get(account = user_account).id
 
             if not User.objects.filter(account = user_account).exists() :
                 return JsonResponse({"Message": "INVALIDE_USER"}, status=401)
 
-            if User.objects.get(account = user_account).password != user_password :
-                return JsonResponse({"Message": "INVALID_USER"}, status=401)
+            hashed_password = user_password.encode("utf-8")
+            saved_password  = User.objects.get(account = user_account).password
+            
+            if not bcrypt.checkpw(hashed_password, saved_password.encode("utf-8")) :
+                return JsonResponse({"Message" : "INVALID_USER"}, status=401)
+
+            access_token = jwt.encode({"user_id" : user_id}, SECRET_KEY, ALGORITHM)
```

Comment:
user의 id를 주는 부분도 위에서 선언한 user 를 활용하면 됩니다.

```suggestion
            access_token = jwt.encode({"user_id" : user.id}, SECRET_KEY, ALGORITHM)
```